### PR TITLE
DDF-1721 certnew-script-for-windows

### PR DESCRIPTION
**Pull request by ahoffer**
CertNew.cmd file is analog of CertNew.sh
for Windows OS

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/392)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/392" target="_blank">ReviewNinja

File: distribution/ddf-common/src/main/resources/etc/certs/CertNew.sh [hunk 1]
```diff
@@ -42,10 +41,11 @@ fi
 
 echo "--IGNORE SLF4J ERRORS"--
 
-RETURNCODE=$(java -cp "$JARFILE" -Djavax.net.ssl.keyStore="$KEYFILE" -Djavax.net.ssl.keyStorePassword="$PASSWORD" -Djavax.net.ssl.keyStoreType="$KEYTYPE" "$CLASSNAME" "$COMMONNAME")
```
Comment (ahoffer) [inline distribution/ddf-common/src/main/resources/etc/certs/CertNew.sh]: Updated the bash script. A copy and paste error from an old version wiped out the correct way to get the return code. I fixed that.

File: distribution/ddf-common/src/main/resources/etc/certs/CertNew.sh [hunk 2]
```diff
@@ -18,7 +18,6 @@ cd `dirname $0`
 PASSWORD="changeit"
 KEYFILE="../keystores/serverKeystore.jks"
 KEYTYPE="JKS"
-CLASSNAME="org.codice.ddf.security.certificate.generator.CertificateCommand"
```
Comment (ahoffer) [inline distribution/ddf-common/src/main/resources/etc/certs/CertNew.sh]: Change the shade plugin configuration to generate an executable JAR file. Class path/name no longer necessary,

File: platform/security/certificate/security-certificate-generator/pom.xml [hunk 3]
```diff
@@ -97,6 +97,11 @@
                             <goal>shade</goal>
                         </goals>
                         <configuration>
+                            <transformers>
```
Comment (ahoffer) [inline platform/security/certificate/security-certificate-generator/pom.xml]: Generate executable JAR.

**Comment by brendan-hofmann** [comment]
:shipit:

**Comment by AzGoalie** [comment]
Did a full bull build and tests passed.
Tested CertNew.cmd on windows and it all worked!
:+1: 

Steps followed:
1. Build it.
2. Copy distribution zip file to Windows and unzip it.
3. Go to <HOME>/etc/cert folder
4. Execute "CertNew mynewcert"
5. Open up <HOME>/etc/keystore/serverKeystore.jks with something like ​*Keystore Explorer*​ and check to see that "mynewcert" is in the keystore.

**Pull request merged**